### ISSUE TITLE
Echo now requiring authentication on incoming requests

### ISSUE THREAD

**Issue by ezimanyi** [spinnaker/spinnaker#2949]
Echo is now configured to require authentication for incoming requests.  This looks to be a result of the change in https://github.com/spinnaker/echo/pull/283.

 Reproduction steps:
```
ezimanyi@spinnaker-dev:~/dev/spinnaker$ curl -H "Content-Type: application/json" -d '{}' http://localhost:8089/; echo
{"timestamp":1529964601103,"status":401,"error":"Unauthorized","message":"Full authentication is required to access this resource"}
```

Before the change, the curl request above would proceed to `HistoryController.saveHistory`.  (Of course the empty payload would throw other validation errors, but this is just to demonstrate the auth issue.)

Effects:
I noticed this in particular because Jenkins triggers were no longer working because `igor'`s call to `echo` to notify it of new builds was failing with the above error.

It looks like the reason this is happening is the following from the [Spring documentation](https://docs.spring.io/spring-boot/docs/current/reference/html/boot-features-security.html):
```If Spring Security is on the classpath, then web applications are secured by default.```

The change to pull `spinnaker.group("fiat")` into echo transitively pulls in spring security, and triggers this authentication check.

Presumably there's a way to disable this check, though I'm not familiar enough with Spring security to know offhand.  Alternatively, it might be possible to replace the `spinnaker.group("fiat")` with something more targeted.

**Comment by ajordens** [comment]
Thanks for filing this ... noticed it on our side and it's on my plate to address. 

Expect to have it fixed by this evening.

**Comment by ezimanyi** [comment]
@ajordens : Awesome, thanks!

**Comment by ajordens** [comment]
Should be fixed by spinnaker/echo#286 ... the `curl` above no longer results in an authentication failure.
